patroni/postgresql.py

```diff
@@ -375,7 +375,10 @@ def reload_config(self, config):
             self._replace_pg_hba()
 
         if conf_changed or hba_changed:
+            logger.info('Configuration items changed, reloading configuration.')
             self.reload()
+        else:
```

Comment:
```diff
- else:
+ elif not pending_restart:
```

Otherwise it will show the message on the next line when you change for example only `max_connections`.